src/VirtualClient/VirtualClient.Dependencies/Packaging/ZypperPackageInstallation.cs

```diff
@@ -135,7 +135,7 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
                 return;
             }
 
-            string formattedArguments = $"--non-interactive install {string.Join(' ', toInstall)}";
+            string formattedArguments = $"--non-interactive install -y {string.Join(' ', toInstall)}";
```

Comment:
Thanks for doing this, is it -y or -n?

Reply:
it's -y! and tested it out for myself on a VM

Reply:
oh sorry I remember I put --non-interactive, then is the -y still needed?

Reply:
yep! it is actually needed in the install exec